#ifndef MAINWINDOW_H
#define MAINWINDOW_H

#include <QMainWindow>
#include <QTextStream>
#include <QSplineSeries>
#include <QChart>
#include <QChartView>
#include <QLineSeries>
#include <QAreaSeries>

#include <QBarSet>
#include <QBarSeries>
#include <QBarCategoryAxis>
#include <QValueAxis>

//#include "models/router.h"

//QT_CHARTS_USE_NAMESPACE

namespace Ui {
class MainWindow;
}

class MainWindow : public QMainWindow
{
    Q_OBJECT

public:
    explicit MainWindow(QWidget *parent = nullptr);
    ~MainWindow();

    void showAreaChart();
    void showLineChart();
    void showBarChart();

private:
    Ui::MainWindow *ui;
};

#endif // MAINWINDOW_H
